### ISSUE TITLE
Tests for analysis modules

### ISSUE THREAD

**Issue by norhh** [ConsenSys/mythril#517]
## Description
write tests for analysis modules

**Comment by b-mueller** [comment]
I suspect that this is indirectly addressed by #513?

**Comment by JoranHonig** [comment]
@b-mueller Yes and no, #513 addresses system tests (testing all the components of mythril), while this is more about component tests of the analysis modules
